publish/tests/fuzzy.py

```diff
@@ -33,3 +33,15 @@ def __init__(self):
 class VersionRe(Re):
     def __init__(self):
         super().__init__(Version.VERSION_REGEX)
+
+
+# TODO: Right now these only validate that the name and description
+# contain at least one non-whitespace character.
+class NameRe(Re):
```

Comment:
I think we should discuss with @dchiquit how far we want to take this RE-based test approach. Now that we're getting so much of our data from Faker, it changes some of my initial assumptions.